internal/pkg/ns1-sdk/controllers/zones.go

```diff
@@ -0,0 +1,27 @@
+package controllers
```

Comment:
I think that the sdk and the api are two different things, as the sdk is a helper to use the ns1 client functions and the api is just one way to use that sdk.
Havent this in mind, shouldnt we have both the controllers and router directories that are used to set up the api in a separate 'official_api' directory?